core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -101,6 +102,7 @@
     private final Map<NodeRef<Parameter>, Expression> parameters;
     private String updateType;
     private Optional<UpdateTarget> target = Optional.empty();
+    private Optional<List<Column>> targetColumns = Optional.empty();
```

Comment:
Optional of collection is an anti-pattern. Is there a distinction between empty optional and empty list in this case? If not, then the optional is not needed